### PR TITLE
PMM-6982 Pass ENABLE_ALERTING

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: pmm/pmm2-api-tests.groovy [hunk 1]
```diff
@@ -3,7 +3,7 @@ void runStaging(String DOCKER_VERSION, CLIENT_VERSION, CLIENTS) {
         string(name: 'DOCKER_VERSION', value: DOCKER_VERSION),
         string(name: 'CLIENT_VERSION', value: CLIENT_VERSION),
         string(name: 'PS_VERSION', value: '5.6'),
-        string(name: 'DOCKER_ENV_VARIABLE', value: '-e PMM_DEBUG=1 -e PERCONA_TEST_SAAS_HOST=check-dev.percona.com:443 -e PERCONA_TEST_CHECKS_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX -e PERCONA_TEST_CHECKS_INTERVAL=10s -e PERCONA_TEST_DBAAS=0'),
+        string(name: 'DOCKER_ENV_VARIABLE', value: '-e PMM_DEBUG=1 -e ENABLE_ALERTING=1 -e PERCONA_TEST_SAAS_HOST=check-dev.percona.com:443 -e PERCONA_TEST_CHECKS_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX -e PERCONA_TEST_CHECKS_INTERVAL=10s -e PERCONA_TEST_DBAAS=0'),
```
Comment (puneet0191) [inline pmm/pmm2-api-tests.groovy:6]: do we need any other parameter to run those tests on api tests docker run?

**Review by puneet0191 — APPROVED** [review]

File: pmm/pmm2-api-tests.groovy (hunk 1 above)
Comment (AlekSi) [reply pmm/pmm2-api-tests.groovy:6]: Started to reply, but then understood that that's incorrect :) One sec

Comment (AlekSi) [reply pmm/pmm2-api-tests.groovy]: "ENABLE_ALERTING" is a "first-class" environment variable like DISABLE_TELEMETRY or DATA_RETENTION. It is disable by default, but probably will be enabled later.

Our API tests actually enable and disable IA

File: pmm/pmm2-api-tests.groovy [hunk 2]
```diff
@@ -111,7 +111,7 @@ pipeline {
         stage('Run API Test') {
             steps {
                 sh '''
-                    sudo docker run -e PMM_SERVER_URL=\${PMM_URL} -e PMM_RUN_UPDATE_TEST=1 -e PMM_RUN_STT_TESTS=0 -e PMM_RUN_IA_TESTS=1 --name ${BUILD_TAG} pmm-api-tests
+                    sudo docker run -e PMM_SERVER_URL=\${PMM_URL} -e PMM_RUN_UPDATE_TEST=1 -e PMM_RUN_STT_TESTS=0 --name ${BUILD_TAG} pmm-api-tests
```
Comment (AlekSi) [inline pmm/pmm2-api-tests.groovy:114]: PMM_RUN_IA_TESTS will be removed from API tests – they will run by default, without a way to disable it

**Comment by AlekSi** [comment]
PTAL

File: pmm/pmm2-api-tests.groovy (hunk 2 above)
Comment (puneet0191) [reply pmm/pmm2-api-tests.groovy:114]: okay understood, so start the server with IA enabled and run those (IA) tests all the time now

**Review by puneet0191 — APPROVED** [review]

**Comment by AlekSi** [comment]
@puneet0191 Please merge – this PR is assigned to you

**Comment by puneet0191** [comment]
definately!

**Pull request merged**